Adding css style to <pre> tags to wrap text on issue description

loganSIO opened the pull request:
@fcayre ici l'ajout du style css dans les balises pre des issues description

loganSIO committed:
Adding css style to <pre> tags to wrap text on issue description

fcayre commented on commown_self_troubleshooting/data/ContractManagement/termination/issue_template.xml:
```diff
@@ -17,7 +17,7 @@
 
         <p t-if="more_info">Informations complémentaires :</p>
-        <pre t-if="more_info"><t t-esc="more_info" /></pre>
+        <pre style="white-space: pre-wrap;"><t t-esc="more_info" /></pre>
```
On voit ici que le code utilisé pour afficher more_info est présent à différents endroits. C'est pas top car : 
1. l'affichage / les fontionnalités ne seront pas les mêmes partout, pour aucune bonne raison (en particulier ici : tu as supprimé le t-if="more_info" qui permet de ne pas afficher de balises vides quand le message était vide)
2. tu dois faire la même modif à plusieurs endroits sans en oublier

Du coup la bonne pratique est plutôt de : 
- créer un mini-template pour afficher le more_info, avec le meilleur des codes possible, donc je pense : 
```
        <p t-if="more_info">Informations complémentaires :</p>
        <div t-if="more_info">
          <pre style="white-space: pre-wrap;"><t t-esc="more_info" /></pre>
        </div>
```
- appeler ce template depuis les différents fichiers issue_template.xml

loganSIO replied:
@fcayre ceci est fait dans le dernier commit.

loganSIO committed:
Templating the more_info section using a white_space style for every issue_template.xml

fcayre commented:
@loganSIO ta modif fait planter l'un des nouveaux tests (tests dans le navigateur) qui attend le texte "informations complémentaires suivantes" et ne le trouve pas, cf. https://github.com/commown/commown-odoo-addons/actions/runs/5832011859/job/15816566354?pr=187#step:7:1939

Pour corriger ça, il suffit d'effacer le mot "suivantes" du fichier commown_self_troubleshooting/static/src/self_troubleshooting_tour.js (l. 219). Je te laisse corriger ta PR et repousser, ce qui devrait rendre les tests contents :)

loganSIO committed:
Removing word 'suivantes' from /static/src/self_troubleshooting_tour.js

loganSIO commented:
> @loganSIO ta modif fait planter l'un des nouveaux tests (tests dans le navigateur) qui attend le texte "informations complémentaires suivantes" et ne le trouve pas, cf. https://github.com/commown/commown-odoo-addons/actions/runs/5832011859/job/15816566354?pr=187#step:7:1939
> 
> Pour corriger ça, il suffit d'effacer le mot "suivantes" du fichier commown_self_troubleshooting/static/src/self_troubleshooting_tour.js (l. 219). Je te laisse corriger ta PR et repousser, ce qui devrait rendre les tests contents :)

Le test a planté à un endroit similaire : "2023-08-20 10:26:34,436 8757 ERROR odoo odoo.addons.commown_self_troubleshooting.tests.test_tour.TestPageFP2: Tour commown_self_troubleshooting_tour_fp2_battery_contact_human failed at step p:contains('informations complémentaires') "

fcayre commented:
> 

LOL oui bon dans le code plus loin on écrit "Informations complémentaires" et on recherche "informations complémentaires"... Comme c'est du jQuery et qu'il n'y a pas de fonction "contains" case-insensitive, on va simplement enlever le "i" du mot "informations" de la ligne 219 du même fichier. Tu ne sembles pas avoir compris comment trouver cette information dans la sortie du test... je te montrerai la prochaine fois qu'on se croisera. Tu corriges et tu repousses ? Tu n'arrives pas à passer ces tests chez toi pour vérifier qu'ils passent avant de pousser ?!

loganSIO committed:
Fixing FP2 battery test

When this test verifies if a p contains "informations complémentaires" it is case sensitive.
So we removed the "i" then it only checks if the p contains the rest of the string.

loganSIO commented:
> 

@fcayre , c'est tout bon, ready pour un merge

loganSIO commented:
Quel boulet j'ai close la MR au lieu de cliquer sur comment !

fcayre commented:
/ocabot merge patch
